README.md

```diff
@@ -5,11 +5,64 @@
 
 
 - [resilience-for-software](#resilience-for-software)
+  - [Why should I invest my time in learning about resilience engineering?](#why-should-i-invest-my-time-in-learning-about-resilience-engineering)
+  - [Where do I start? What is resilience engineering?](#where-do-i-start-what-is-resilience-engineering)
+  - [What is the relationship between resilience engineering and DevOps/SRE?](#what-is-the-relationship-between-resilience-engineering-and-devopssre)
+  - [I’m intrigued, what’s next?](#im-intrigued-whats-next)
+  - [What should I do if I have other questions?](#what-should-i-do-if-i-have-other-questions)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 <!-- prettier-ignore-end -->
 
 # resilience-for-software
 
-Introduction to resilience engineering concepts for software engineers
+The intent of this document is to provide a lightweight introduction to and motivate
+further engagement with resilience engineering through a “narrative FAQ”.
+
+## Why should I invest my time in learning about resilience engineering?
```

Comment:
The concept is good, but I think the wording could be tweaked. Maybe "What makes RE worth investing time in?" or "What's the return on investing time in RE?". Less about individual self so that it doesn't so much a "what's in it for me?" vibe.